src/amuse/test/suite/compile_tests/test_c_implementation_simplified.py

```diff
@@ -597,6 +598,7 @@ def test30(self):
           "got multiple values for argument 'i' of method dummy_3_int")
         instance.stop()
 
+    @pytest.mark.skip
```

Comment:
hmmm. this test should work...

Reply:
Yes - but it doesn't... same for test31 in fortran implementation.

Reply:
no idea why yet, but these tests hang.

Reply:
it should, but it doesn't. test31 in test_c_implementation_simplified.py and test_fortran_implementation is problematic for some reason, both on travis and for me locally (macOS + Linux). I don't know why...